src/main/java/net/sf/jabref/logic/formatter/bibtexfields/NormalizeNamesFormatter.java

```diff
@@ -36,6 +44,75 @@ public String getKey() {
 
     @Override
     public String format(String value) {
+        Objects.requireNonNull(value);
```

Comment:
@oscargus It doesen't matter in which class we put the new code. It will first be executed anyway. I think it belongs to AuthorList parse, because the other methods in authorlist will always be executed afterwards. Like this we have a better reading flow.